commerce-add-ons/payment-providers/quickpay/configuring-quickpay.md

```diff
@@ -15,3 +15,34 @@ If you haven't done so yet, head over to the [QuickPay portal](https://manage.qu
 Or [login to the QuickPay portal](https://manage.quickpay.net/) if you already have an account.
 
 ![The login form to the QuickPay portal.](../media/quickpay/quickpay\_signin.png)
+
+## Step 2: API Keys
+
+In order for Umbraco Commerce to communicate with QuickPay securely we need to retrieve a series of API keys used for authentication.
+
+The keys can be found under **Settings > Integration** in the QuickPay portal.
+
+You will need the following keys:
+
+- Private key
+- Api key of the API user
+- Merchant id
+- Agreement id
```

Comment:
I have checked the image shared below however it seems the mentioned content does not match with the ones from the image. Could an idea be in this case to write for example instead of "private key" - "Private key of the Account name - ", "Api Keyof the Payment Window", "Api key of User mail" etc. 💡 
![image](https://github.com/umbraco/UmbracoDocs/assets/83591955/4f7b90b1-2def-4952-8aa0-9929a5d3a918)

Reply:
Oh I see! In this case my recommendation would be to change the previous sentences like so "There are 4 Keys and they can be found under **Settings > Integration** in the QuickPay portal.

For example, based on the image bellow you will need the following keys:"